<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>gai</class>
 <widget class="QWidget" name="gai">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1234</width>
    <height>660</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
background:url(:/res/LW_12.jpg);</string>
  </property>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>10</y>
     <width>121</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
color:white;
font-size:32px;
font-family:Microsoft JhengHei;
background:transparent;
}</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:24pt; font-weight:600; color:#aaffff;&quot;&gt;书名：&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>80</y>
     <width>231</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
color:white;
font-size:32px;
font-family:Microsoft JhengHei;
background:transparent;
}</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:24pt; font-weight:600; color:#aaffff;&quot;&gt;图书isbn：&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>150</y>
     <width>231</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
color:white;
font-size:32px;
font-family:Microsoft JhengHei;
background:transparent;
}</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:24pt; font-weight:600; color:#aaffff;&quot;&gt;作者：&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>220</y>
     <width>231</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
color:white;
font-size:32px;
font-family:Microsoft JhengHei;
background:transparent;
}</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:24pt; font-weight:600; color:#aaffff;&quot;&gt;出版社：&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>290</y>
     <width>231</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
color:white;
font-size:32px;
font-family:Microsoft JhengHei;
background:transparent;
}</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:24pt; font-weight:600; color:#aaffff;&quot;&gt;简介：&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>10</y>
     <width>231</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
color:white;
font-size:32px;
font-family:Microsoft JhengHei;
background:transparent;
}</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:24pt; font-weight:600; color:#aaffff;&quot;&gt;类别：&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>940</x>
     <y>20</y>
     <width>171</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QComboBox{
font: 75 20pt &quot;Ink Free&quot;;
background:rgba(0,0,0,0);
border-radius:25px;
}



/* 下拉后，整个下拉窗体越过每项的样式 */
QComboBox QAbstractItemView::item:hover {
    font: 75 20pt &quot;Ink Free&quot;;
background:rgba(0,0,0,0);
border-radius:25px;  /* 整个下拉窗体越过每项的背景色 */
}


QComboBox QAbstractItemView::item:selected {
    font: 75 20pt &quot;Ink Free&quot;;
background:rgba(0,0,0,0);
border-radius:25px;
}</string>
   </property>
   <item>
    <property name="text">
     <string>Linux</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>深度学习</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>机器学习</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>英语</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>80</y>
     <width>231</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
color:white;
font-size:32px;
font-family:Microsoft JhengHei;
background:transparent;
}</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:24pt; font-weight:600; color:#aaffff;&quot;&gt;价格：&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>10</y>
     <width>471</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit{
background:transparent;
border:none;
color:#FFFFFF;
font-size:25px;
font-color:#FFFFFF;
font-family:Microsoft JhengHei;
border-bottom:1px solid #717072;
}</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_2">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>80</y>
     <width>471</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit{
background:transparent;
border:none;
color:#FFFFFF;
font-size:25px;
font-family:Microsoft JhengHei;
border-bottom:1px solid #717072;
}</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_3">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>150</y>
     <width>471</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit{
background:transparent;
border:none;
color:#FFFFFF;
font-size:25px;
font-family:Microsoft JhengHei;
border-bottom:1px solid #717072;
}</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_4">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>220</y>
     <width>471</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit{
background:transparent;
border:none;
color:#FFFFFF;
font-size:25px;
font-family:Microsoft JhengHei;
border-bottom:1px solid #717072;
}</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_5">
   <property name="geometry">
    <rect>
     <x>930</x>
     <y>80</y>
     <width>221</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit{
background:transparent;
border:none;
color:#FFFFFF;
font-size:25px;
font-family:Microsoft JhengHei;
border-bottom:1px solid #717072;
}</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_6">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>300</y>
     <width>871</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit{
background:transparent;
border:none;
color:#FFFFFF;
font-size:25px;
font-family:Microsoft JhengHei;
border-bottom:1px solid #717072;
}</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>500</y>
     <width>181</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
	font: 25pt &quot;Arial&quot;;
background:rgba(237, 255, 167,0.70);
border-radius:26px;
}
QPushButton:hover{
background:rgba(121,0,255,0.20);
border-radius:26px;

}</string>
   </property>
   <property name="text">
    <string>提  交</string>
   </property>
   <property name="iconSize">
    <size>
     <width>50</width>
     <height>50</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>880</x>
     <y>500</y>
     <width>181</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
	font: 25pt &quot;Arial&quot;;
background:rgba(237, 255, 167,0.70);
border-radius:26px;
}
QPushButton:hover{
background:rgba(121,0,255,0.20);
border-radius:26px;

}</string>
   </property>
   <property name="text">
    <string>取  消</string>
   </property>
   <property name="iconSize">
    <size>
     <width>50</width>
     <height>50</height>
    </size>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
